sciplot/PlotBase.hpp

```diff
@@ -123,6 +128,9 @@ class PlotBase
     /// Write the current plot data to the data file.
     auto savePlotData() const -> void;
 
+    /// Toggle the use of custom draw commands (disabled by default).
```

Comment:
/`// Toggle the use of custom draw commands (disabled by default) this will replace all regular draw* command by the string passed here. You can set custom draw command using drawCustom().`

Reply:
This explanation would not be fitting entirely. The draw* commands are not replaced, but rather not loaded by `repr()` for the generated script. The `m_drawSpecs` vector remains the same and can be used again after toggling `m_usecustomdrawcmds` to false.

So how about this:
```
/// Toggle the use of custom draw commands (disabled by default).
/// You can set custom draw commands using drawCustom().
/// @note This will disable (but not remove) all regular draw* commands.

Reply:
But the draw command are replaced in the file?!

Reply:
That's correct. We might be talking about different scopes here, I only referred to the internal container for commands, not to the resulting script.

So would you prefer having something like this?
```
/// Toggle the use of custom draw commands (disabled by default).
/// You can set custom draw commands using drawCustom().
/// @note This will remove all regular draw* commands from the generated script.
```

Reply:
Think as if someone is using you stuff. What does he/she need to know?
How about:

> ...this will replace all regular draw* commands in the generated script by the string passed here. You can set custom draw command using drawCustom().

Reply:
Sounds reasonable, I'll include this one.